Gemfile.tt

```diff
@@ -43,12 +43,12 @@ group :development, :test do
   gem 'rubocop', require: false
   gem 'rubocop-rails', require: false
 
-  gem 'pronto'
-  gem 'pronto-rubocop', require: false
-  gem 'pronto-flay', require: false # analyzes code for structural similarities
-  gem 'pronto-brakeman', require: false # analyzes code for security vulnerabilities
-  gem 'pronto-rails_best_practices', require: false
-  gem 'pronto-reek', require: false
+  gem 'danger' # Automated code review.
+  gem 'danger-rubocop' # Analyze code regarding best practices.
```

Comment:
Should we change this description?
I think we're copying from below

Reply:
Added new commit!